packages/relay/index.js

```diff
@@ -1,11 +1,14 @@
 // @flow strict
 
-/* eslint-disable no-restricted-imports */
+/* eslint-disable no-restricted-imports, no-unused-vars */
+import { type RelayRefetchProp } from 'react-relay';
```

Comment:
Does that really work?

Reply:
I suppose that it works 
<img width="796" alt="screen shot 2019-02-07 at 17 19 50" src="https://user-images.githubusercontent.com/6354326/52425614-a7643c00-2afc-11e9-8965-39d3e99cc5b6.png">